test/commands/code-analyzer/config.test.ts

```diff
@@ -28,184 +28,188 @@ describe('`code-analyzer config` tests', () => {
+			expect(receivedActionInput).toHaveProperty('config-file', inputValue);
+		});
 
-			it('Is unused if not directly specified', async () => {
-				await ConfigCommand.run([]);
-				expect(executeSpy).toHaveBeenCalled();
-				expect(receivedActionInput['config-file']).toBeUndefined();
-			});
+		it('Rejects non-existent file', async () => {
+			const inputValue = 'definitelyFakeFile.json';
+			const executionPromise = ConfigCommand.run(['--config-file', inputValue]);
+			await expect(executionPromise).rejects.toThrow(`No file found at ${inputValue}`);
+			expect(executeSpy).not.toHaveBeenCalled();
+		});
 
-			it('Can only be supplied once', async () => {
-				const inputValue1 = 'package.json';
-				const inputValue2 = 'LICENSE';
-				const executionPromise = ConfigCommand.run(['--config-file', inputValue1, '--config-file', inputValue2]);
-				await expect(executionPromise).rejects.toThrow(`Flag --config-file can only be specified once`);
-				expect(executeSpy).not.toHaveBeenCalled();
-			});
+		it('Is unused if not directly specified', async () => {
+			await ConfigCommand.run([]);
+			expect(executeSpy).toHaveBeenCalled();
+			expect(receivedActionInput['config-file']).toBeUndefined();
+		});
 
-			it('Can be referenced by its shortname, -c', async () => {
-				const inputValue = 'package.json';
-				await ConfigCommand.run(['-c', inputValue]);
-				expect(executeSpy).toHaveBeenCalled();
-				expect(receivedActionInput).toHaveProperty('config-file', inputValue);
-			});
+		it('Can only be supplied once', async () => {
+			const inputValue1 = 'package.json';
+			const inputValue2 = 'LICENSE';
+			const executionPromise = ConfigCommand.run(['--config-file', inputValue1, '--config-file', inputValue2]);
+			await expect(executionPromise).rejects.toThrow(`Flag --config-file can only be specified once`);
+			expect(executeSpy).not.toHaveBeenCalled();
 		});
 
-		describe('--workspace', () => {
-			it('Can be supplied once with a single value', async () => {
-				const inputValue = './somedirectory';
-				await ConfigCommand.run(['--workspace', inputValue]);
-				expect(executeSpy).toHaveBeenCalled();
-				expect(receivedActionInput).toHaveProperty('workspace', [inputValue]);
-			});
+		it('Can be referenced by its shortname, -c', async () => {
+			const inputValue = 'package.json';
+			await ConfigCommand.run(['-c', inputValue]);
+			expect(executeSpy).toHaveBeenCalled();
+			expect(receivedActionInput).toHaveProperty('config-file', inputValue);
+		});
+	});
 
-			it('Can be supplied once with multiple comma-separated values', async () => {
-				const inputValue =['./somedirectory', './someotherdirectory'];
-				await ConfigCommand.run(['--workspace', inputValue.join(',')]);
-				expect(executeSpy).toHaveBeenCalled();
-				expect(receivedActionInput).toHaveProperty('workspace', inputValue);
-			});
+	describe.each([
+		{flag: '--workspace', shortflag: '-w', property: 'workspace'},
+		{flag: '--target', shortflag: '-t', property: 'target'}
```

Comment:
Since `--target` and `--workspace` share all the same constraints and expectations on a flag-level, I just made them variants of the same `describe.each` call.